src/test/java/org/folio/rest/api/ItemStorageTest.java

```diff
@@ -2779,4 +2821,19 @@ private Boolean isBefore(String dateTime, String secondDateTime) {
       .map(IndividualResource::getId)
       .collect(Collectors.toList());
   }
+
+  private static void assertItemStatusIs(JsonObject item, Status.Name status) {
+    assertThat(item.getJsonObject("status").getString("name"), is(status.value()));
+  }
+
+  private Response checkOutItem(String itemId) {
+    CompletableFuture<Response> postCompleted = new CompletableFuture<>();
+    client.post(itemsStorageUrl("/" + itemId + "/check-out"), null,
+      StorageTestSuite.TENANT_ID, ResponseHandler.empty(postCompleted));
+    try {
+      return postCompleted.get(5, TimeUnit.SECONDS);
+    } catch (InterruptedException | ExecutionException | TimeoutException e) {
+      throw new RuntimeException(e);
+    }
```

Comment:
This can be replaced by the `SneakyThrows` annotation from lombok

Reply:
Replaced, thanks